embrace-android-sdk/src/androidTest/java/io/embrace/android/embracesdk/ndk/EmbraceNativeLayerTest.kt

```diff
@@ -0,0 +1,26 @@
+package io.embrace.android.embracesdk.ndk
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+internal class EmbraceNativeLayerTest {
+
+    companion object {
+        init {
+            System.loadLibrary("embrace-native")
+            System.loadLibrary("embrace-native-test")
+        }
+    }
+
+    external fun run(): Int
+
+    @Test
+    fun testNativeLayer() {
+        val status = run()
+        assertEquals(
```

Comment:
why do we expect an error here?

Reply:
This is perhaps a little confusing as it's on multiple lines, but the first param is the message used by JUnit if the test fails. We're asserting that the status is 0 (normal termination) as the native code will return 1 if a test failed. I just wanted the error message to be a bit clearer when failures do happen

Reply:
ohhhh never used assertEquals like that, sorry! 🤦